app/javascript/components/activity/ProductTotals.vue

```diff
@@ -71,6 +79,7 @@ export default {
       let params = {user: this.user.id, paid_with_cash: this.user.paid_with_cash, paid_with_pin: this.user.paid_with_pin};
       this.$http.get('/activities/'+this.activity+'/product_totals', { params }).then((response) => {
         this.orderTotals = response.body;
+        this.totalAmount = this.orderTotals.reduce((a, b) => a + parseFloat(b.price), 0.0);
```

Comment:
i am getting a warning trying to run this 
![image](https://github.com/user-attachments/assets/00f9ad3c-4e99-4f03-901b-3a7e2514f483)